docs/visualization/wcsaxes/index.rst

```diff
@@ -0,0 +1,75 @@
+.. _wcsaxes:
+
+*********************************************
+Making plots with world coordinates (WCSAxes)
+*********************************************
+
+WCSAxes is a framework for making plots of Astronomical data in Matplotlib. It
+was previously distributed as a standalone package, but is now included in
+:ref:`astropy.visualization <astropy-visualization>`.
+
+Compared to packages such as `APLpy <http://aplpy.readthedocs.io>`__, WCSAxes is
+a framework that is closer to the Matplotlib API and allows more advanced plots
+to be made, but on the other hand requires more code for simple plots. APLpy
+will in future be updated to make use of WCSAxes and provide a simpler
+interface for making common plots.
+
+Getting started
+===============
+
+The following is a simple example of plotting an image with the WCSAxes
+package, and overlaying the coordinate grid from the image as well as an
+equatorial coordinate grid.
+
+.. plot::
```

Comment:
I wrote a whole comment to rant about the pyplot interface but in the end I'm just going to update the docs to make sure I mention both and in most places try and make it so it's not too different anyway...

Just a pedantic point - I'm not using the OO interface, I'm using the 'semi-OO' interface (there are three interfaces in Matplotlib!). The full OO interface is indeed a bit too confusing.